src/components/shared/Dropdown.js

```diff
@@ -33,12 +34,12 @@ const Dropdown = React.createClass({
   },
 
   renderButton() {
-    return dom.button(
+    return dom.div(
```

Comment:
is there a reason to change this to a div ? From an accessibility point of view, it would be really nice to have this still as a button (because it implies the user can act on it, whereas a div is just a container).
It's not the scope of your PR, but it would be even nicer if we could have an `aria-haspopup` attribute (https://msdn.microsoft.com/fr-fr/library/windows/apps/hh465704.aspx) on this :)